tests/weekly_tests/vivado_ooc.yaml

```diff
@@ -3,4 +3,4 @@ design_dirs:
 
 flow: vivado_ooc
 
-synth: "-flatten_hierarchy full"
\ No newline at end of file
+synth_options: "-flatten_hierarchy full"
```

Comment:
I think this is added by the flow already, and not required here.  Can you double check?

Reply:
I'm not actually sure on this one, vivado_ooc.py doesn't specify it's synth_options like vivado_phys_netlist_cmp.py does and I don't know how else to check.